Source/WTF/wtf/PlatformUse.h

```diff
@@ -440,6 +440,11 @@
 #define USE_SANDBOX_VERSION_3 1
 #endif
 
+// We already enable ExtensionKit when building with the internal SDK. Also enable it in Open Source builds.
+#if !defined(USE_EXTENSIONKIT) && PLATFORM(IOS) && __IPHONE_OS_VERSION_MAX_ALLOWED >= 170400 && !USE(APPLE_INTERNAL_SDK)
```

Comment:
Why does this need `&& !USE(APPLE_INTERNAL_SDK)`? If `USE_EXTENSIONKIT` is defined for APPLE_INTERNAL_SDK else where, can we just drop that and use  `!defined(USE_EXTENSIONKIT) && PLATFORM(IOS) && __IPHONE_OS_VERSION_MAX_ALLOWED >= 170400` here?

Reply:
Per explained to me offline that this is currently conditionally enabled with internal SDK.